pkg/minikube/driver/version.go

```diff
@@ -0,0 +1,31 @@
+package driver
+
+import (
+	"github.com/blang/semver"
+	"github.com/golang/glog"
+)
+
+// minHyperkitVersion is the minimum version of the minikube hyperkit driver compatible with the current minikube code
+var minHyperkitVersion semver.Version
+
+const minHyperkitVersionStr = "1.10.0"
+
+func init() {
+	v, err := semver.New(minHyperkitVersionStr)
+	if err != nil {
+		glog.Fatalf("Failed to parse the hyperkit driver version: %v", err)
```

Comment:
lets not fatal and if severErr is not nil lets just download the latest version of driver as we do today.

Reply:
Makes sense. Fixed.